dist/pinny.js

```diff
@@ -467,6 +470,26 @@
             return $(document.activeElement);
         },
 
+        _isWithin: function($element, selector) {
+            return ($element.closest('.' + selector).length) ? true : false;
```

Comment:
Will add $.isChildOf() to selector-utils. Will also externalize _activeElement to selector-utils lib.